.env

```diff
@@ -1 +1,2 @@
-REACT_APP_BASEURL = "https://reactive-herald-api.herokuapp.com/api/v1"
\ No newline at end of file
+REACT_APP_BASEURL = "https://reactive-herald-api.herokuapp.com/api/v1"
```

Comment:
add both .env files to `.gitignore`, everyone needs to have this locally. Share it in the project channel.

Reply:
True, fixed